openpype/hosts/maya/plugins/create/convert_legacy.py

```diff
@@ -135,6 +138,18 @@ def _convert_per_renderlayer(self, instance_nodes, data, creator):
             # "rendering" family being converted to "renderlayer" family)
             original_data["family"] = creator.family
 
+            # recreate subset name as without it would be
+            # `renderingMain` vs correct `renderMain`
+            project_name = self.create_context.get_current_project_name()
+            asset_doc = get_asset_by_name(project_name,
+                                          original_data["asset"])
+            subset_name = get_subset_name("render",
+                                          original_data["variant"],
+                                          data["task"],
+                                          asset_doc,
+                                          project_name)
+            original_data["subset"] = subset_name
+
```

Comment:
If [this](https://github.com/ynput/OpenPype/pull/5442/files#r1291614953) is true then this logic can all be removed.
```suggestion
```

Reply:
No, still needed, without it it will produce `renderingMain` subset name

Reply:
Somehow that sounds like a different bug then - I'd expect the logic to retrieve the subset name from the creator - instead of making one up randomly?